### DISCUSSION TITLE
Is within_no_turns header file needed?

### DISCUSSION THREAD

**Discussion by vissarion** [boostorg/geometry#1095] [Q&A]
The following file https://github.com/boostorg/geometry/blame/develop/include/boost/geometry/algorithms/detail/within/within_no_turns.hpp is not used by the library. It implements some spacial cases of within and is rather old. So the question is: is it needed?

**Comment by barendgehrels** [comment]
It's indeed very old. I don't remember. @awulkiew can we remove it?
